apps/examples/st_things/Kconfig

```diff
@@ -19,33 +18,45 @@ choice
 
 config EXAMPLES_ST_THINGS_LIGHT
 	bool "LIGHT"
-	select IOTBUS_GPIO
 	---help---
 		This is a Light Sample.
 
 config EXAMPLES_ST_THINGS_BLINK
 	bool "BLINK"
+	select IOTBUS
 	select IOTBUS_GPIO
 	---help---
 		This is a Blink Sample.
 
 config EXAMPLES_ST_THINGS_MSG_HANDLING
 	bool "Message Handling Example"
+	select IOTBUS
 	select IOTBUS_GPIO
 	---help---
 		This is a Message handling Sample.
 
 endchoice
 
+config DEMO_LED
```

Comment:
Why did you make new config for sample.
As you know there is EXAMPLE_ST_THINGS_LIGHT. 
It is not good to make new one. 

Could you please consider to use 'EXAMPLE_ST_XX'?

Reply:
This PR is just used for demo with external LEDs easily.
I specify GPIO pins for LEDs and buttons in os/board/esp32_devkit/src/esp32_boot.c
and iotbus does not work on my side, so I also create new thread to scan button status...
Anyway, this is not a formal PR, I will close it later.
Thanks for your review :)